test/astable_flag.jl

```diff
@@ -0,0 +1,178 @@
+module TestAsTableFlag
+
+using Test
+using DataFramesMeta
+using Statistics
+
+const ≅ = isequal
+
+@testset "@astable with just assignments" begin
+    df = DataFrame(a = 1, b = 2)
+
+    d = @rtransform df @astable begin
+        :x = 1
+
```

Comment:
newline is not needed. Also I am not clear why you add `nothing` here and below? Does it change anything?